lib/jsurl.js

```diff
@@ -35,6 +35,8 @@
 			});
 		}
 
+		var tmpAry = [];
```

Comment:
I would prefer to have this variable initialized only in the blocks that use it, to save the allocation of an array when tmpAry is not used.